aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -275,9 +290,15 @@ module aptos_framework::account {
     /// 1. During normal key rotation via `rotate_authentication_key` or `rotate_authentication_key_call`
     /// 2. During resource account initialization so that no private key can control the resource account
     /// 3. During multisig_v2 account creation
-    public(friend) fun rotate_authentication_key_internal(account: &signer, new_auth_key: vector<u8>) acquires Account {
+    public(friend) fun rotate_authentication_key_internal(
+        account: &signer,
+        new_auth_key: vector<u8>
+    ) acquires Account {
         let addr = signer::address_of(account);
-        assert!(exists_at(addr), error::not_found(EACCOUNT_DOES_NOT_EXIST));
+        assert!(
+            exists_at(addr),
+            error::not_found(EACCOUNT_DOES_NOT_EXIST)
```

Comment:
Think we should have a comma at the end of every break line. i.e.
```
assert!(
    exists_at(addr),
    error::not_found(EACCOUNT_DOES_NOT_EXIST),
);
```

Reply:
fixed at https://github.com/movebit/movefmt/commit/9ca658736a254a5696676f95c8da7b498a90281c